core/src/avm2/globals/boolean.rs

```diff
@@ -150,5 +150,12 @@ pub fn create_class<'gc>(activation: &mut Activation<'_, 'gc>) -> GcCell<'gc, Cl
         AS3_INSTANCE_METHODS,
     );
 
+    const CONSTANTS: &[(&str, u32)] = &[("length", 1)];
+    write.define_constant_uint_class_traits(
```

Comment:
This should be `define_constant_int_class_traits` (and the above line should use `&[(&str, i32)]`).

Reply:
Very good catch and thank you very much, I should have been more careful with the types, my knowledge of types in ActionScript is very limited. (I also got a bit confused by https://github.com/ruffle-rs/ruffle/pull/10914#discussion_r1182597426  with `uint`).

I have opened all the types using FFDEC in "playerglobal32_0.swc", and I can confirm that they are all `int` (apart from `Namespace`, where the type has not been set (FFDEC says "null" for the type in P-code source), but I have changed it to `int` there as well).